docs/UserGuide.md

```diff
@@ -87,21 +87,24 @@ Format: `help`
 
 ### Adding a guest: `add`
 
-Adds a guest to the guest book.
+Adds a guest to the guest book. This is akin to checking a guest in with their personal and room details.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`
```

Comment:
Should we use PHONE instead of PHONE_NUMBER for consistency below?

Reply:
Yes, edited!